content/articles/How-to-start-up-with-Socket-Programming-with-Java/index.md

```diff
@@ -29,45 +29,46 @@ The client must know two information about the server:
 1.IP address of Server and 
 2.Port number
 
->Port number 0-1024 are reserved for administrative purposes(e,g 21 for FTP,23 for Telnet,25 for email and 80 for HTTP).
+>Port number 0-1024 are reserved for administrative purposes(e,g 21 for FTP,23 for Telnet,25 for email and 80 for HTTP).In our program we will use port number 5000.
 
-This happens when a TCP connection is being used between two machines:-
+Whenever we create a socket connection in Java, the following steps take place:-
 
 1. server creates a ServerSocket object, to indicate which port number our communication is to occur. If an I/O  error occurs, exception handling mechanisms are implemented; **Throws** an **I/OException** or a **try** **catch** block is used.
 2. The accept() function of the ServerSocket class is called by the server. This method waits for a client to connect to the server on a specific port before returning.
 3. A client builds an object in the socket, giving the name of the server and the connecting port number as the server awaits.
 4. The Socket Class builder will try to connect the client to the server and port number provided. The client now has a socket object that, if communication is established, may connect with the server.
```

Comment:
Change:

>The Socket Class builder will try to connect the client to the server and port number provided. The client now has a socket object that, if communication is established, may connect with the server.

To:

>The Socket class constructor will try to connect the client to the server and port number provided.

Fixing a capitalization error and changing **builder** to **constructor** to be more precise.